.github/workflows/release.yml

```diff
@@ -33,17 +33,13 @@ jobs:
                     fetch-depth: 0
             -   uses: ./.github/actions/setup
 
-            -   name: bump version
-                run: yarn nx affected --target=bump-version --base=latest-release --parallel
-                env:
-                    GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
             -   name: build
                 run: yarn nx affected:build --parallel --production
 
-            -   name: publish to npm
-                run: npx nx affected --target=publish-npm --base=latest-release --parallel
+            -   name: bump version
+                run: yarn nx affected --target=bump-version --base=latest-release --parallel
```

Comment:
Note that we had a problem with yarn and npm publish, that's why we use `npx` instead.

Reply:
A problem with the npm auth?

I faced the same problem before!
https://github.com/yarnpkg/yarn/issues/5683

Reply:
Applyed